src/main/java/com/game/Battleship/controllers/BoardController.java

```diff
@@ -0,0 +1,16 @@
+package com.game.Battleship.controllers;
+
+import com.game.Battleship.models.Board;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BoardController {
+
+    @GetMapping("/board")
+    public Board board(){
```

Comment:
Yowzers there is a lot of board happening here. Is there maybe a better way you can name this method that speaks to either what is being returned or the action that it is performing?

Reply:
Done